core/src/banking_stage/transaction_scheduler/prio_graph_scheduler.rs

```diff
@@ -568,20 +565,12 @@ mod tests {
             let id = TransactionId::new(index as u64);
             let transaction =
                 prioritized_tranfers(from_keypair.borrow(), to_pubkeys, lamports, priority);
-            let transaction_cost = CostModel::calculate_cost(&transaction, &FeatureSet::default());
+            let cost = 5000;
             let transaction_ttl = SanitizedTransactionTTL {
                 transaction,
                 max_age_slot: Slot::MAX,
             };
-            container.insert_new_transaction(
-                id,
-                transaction_ttl,
-                TransactionPriorityDetails {
-                    priority,
```

Comment:
how about renaming this to `compute_unit_price` in this pr or as a prep pr? two things will be referred to as `priority` after this pr, which is confusing.

Reply:
also, rename the struct to `ComputeBudgetDetails` to remove `Priority`.